stack.yaml

```diff
@@ -10,6 +10,7 @@ packages:
 extra-deps:
 - sequence-0.9.8
 - tasty-hedgehog-1.0.0.1 # Needed due to https://github.com/qfpl/tasty-hedgehog/issues/30
+- Unique-0.4.7.6
```

Comment:
Why a specific version here, and not in `cs-ledger.cabal`?

Reply:
Done.